src/main/java/com/vaadin/recipes/recipe/alwayseditablelabel/AlwaysEditableLabel.java

```diff
@@ -0,0 +1,22 @@
+package com.vaadin.recipes.recipe.alwayseditablelabel;
+
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.router.Route;
+import com.vaadin.recipes.recipe.Metadata;
+import com.vaadin.recipes.recipe.Recipe;
+import com.vaadin.recipes.recipe.Tag;
+
+@Route("always-editable-label")
+@Metadata(
+        howdoI = "Create an always-editable Label",
+        description = "Create a Label component that is always in editable mode.",
+        tags = { Tag.JAVA })
+public class AlwaysEditableLabel extends Recipe {
+
+    public AlwaysEditableLabel() {
+        Label label = new Label("I'm an editable label ... ");
+        label.getElement().setAttribute("contenteditable", true);
```

Comment:
This sounds a little bit simplistic as if you have an editor you always want a "value changed" event so you can update the value somewhere

Reply:
Hmm ... I get your point. In that case, I guess there is no point in having #99 as a separate "how to" since its implementation would simply be identical to that of #95 (except that #99 would require listening to a single-click instead of a double-click). Or do you envision a different use case here that would require a rather different implementation?

Reply:
It would be a slightly different case if you do not see a visual ”blink” when clicking but instead can click at any position and directly add/change some text. So visually like what you have but with a way to get the new value

Reply:
http://jsfiddle.net/ch6yn/2691/ indicates that an "input" event is at least fired for "contenteditable". Not sure if it works in all browsers and if "change" or similar events are also fired